packages/example-hello-world/src/application.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-hello-world
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application} from '@loopback/core';
+import {RestApplication, RestServer} from '@loopback/rest';
+
+export class HelloWorldApplication extends RestApplication {
+  constructor() {
+    super();
+    this.handler((sequence, request, response) => {
```

Comment:
This is the critical setup for `example-hello-work`. Please add some comments to convey the message.